### ISSUE TITLE
php-xml missing

### ISSUE THREAD

**Issue by stefaweb** [mailwatch/mailwatch-install-script#35]
Hello @Skywalker-11!

I tried the script with postfix/apache/mariadb.

When we run the script, it ask if we want use RCP (and say that php-xml will be installed).

I answered NO to the question.

After the install, when I go to the GUI, the system says "no php-xml" present and the page stay blank without le GUI login.

I have to install php-xml manually and reload apache.

**Comment by Skywalker-11** [comment]
Which OS?

**Comment by stefaweb** [comment]
Debian Stretch out of the box.

**Comment by Skywalker-11** [comment]
Weird. I just tested it and it succesfully installed the php-xml and php7.0-xml packages. Does it help to restart apache?

**Comment by stefaweb** [comment]
Tried again.

In the GUI : `The PHP extension xml is missing`

I selected:

```
postfix
apache
no RCP
no LDAP
a2dissite 000-default
a2ensite mailwatch
systemctl reload apache2
```

No php-xml and php7.0-xml packages installed.

No luke. ;)

**Comment by Skywalker-11** [comment]
RPC (here aka. php-xml) has to be installed. When I first wrote the install script it was for php5 (xml included in core package). 

With php7 it is neccessary not only for rpc but eg. utf8_decode is part of that package. So it is mandatory to install it.

**Comment by stefaweb** [comment]
Solved with #38.
